Client/src/App.js

```diff
@@ -1,7 +1,10 @@
 import React, {useState, useEffect} from 'react';
 import logo from './logo.svg';
 import './App.css';
-
+import Title from "./Components/Title.js";
+import Splash from "./Components/Splash.js"
+import Blurb from "./Components/Blurb.js"
+import DataCell from "./Components/DataCell.js"
 function App() {
   const[apiResponse, setApiResponse] = useState("");
 
```

Comment:
Could you update the directory structure and create a directory for each component and put the js & css file in there? The way it looks now it will get cluttered really quickly.